### DISCUSSION TITLE
Email notifications for new users

### DISCUSSION THREAD

**Discussion by louiswol94** [directus/api#2389] [Q&A]
Hi everyone,

What is the best way to let new users that are created to receive an email notification about their account with a password etc so that they can log in?

Directus v8.8.1

Is there any way to listen to new users being inserted and running a custom hook? (Something like 'table.insert.directus_users')

This is a lot like "Invite user" on the latest version of Directus.

I just don't want to add a password when creating a user, and send them the password manually.

Kind regards

**Comment by rijkvanzanten** [comment]
You should be able to use a hook for that 👍🏻 

https://v8.docs.directus.io/extensions/hooks.html#hooks

**Reply by louiswol94** [reply]
Thanks for the response @rijkvanzanten 

'item.create.users'?

I tried several without success...

**Reply by rijkvanzanten** [reply]
The collection is `directus_users`, so `item.create.directus_users` might work

**Reply by louiswol94** [reply]
Thanks! :-D
